src/utils/index.js

```diff
@@ -1,12 +1,13 @@
-// This file contains utility functions that are exposed game-side
+/** This file contains utility functions that are exposed game-side */
 
 /**
   * Choose a random number between a and b.
   * @example randRange([a, b], bipolar)
   * @example randRange(a, b, bipolar)
-  * @arg {number} a
-  * @arg {number} b
-  * @arg {boolean} [bipolar] - If true, then the returned value will be inverted half of the time
+  * @global
+  * @arg     {number}  a
+  * @arg     {number}  b
+  * @arg     {boolean} [bipolar=false] Should the returned value will be inverted half of the time
   * @returns {number}
   */
 randRange = function(a, b, bipolar) {
```

Comment:
Here are the handy util functions I carry around with me to accomplish this:
`function rollFloat (n, m) { return n + random() * (m - n); };`
`function rollInt (n, m) { return ~~(n + random() * (1 + m - n)); };`